src/test/java/io/confluent/connect/jdbc/source/JdbcSourceTaskTestBase.java

```diff
@@ -34,9 +34,16 @@
 
   protected static String SINGLE_TABLE_NAME = "test";
   protected static Map<String, Object> SINGLE_TABLE_PARTITION = new HashMap<>();
+  protected static Map<String, Object> SINGLE_TABLE_PARTITION_WITH_VERSION = new HashMap<>();
 
   static {
     SINGLE_TABLE_PARTITION.put(JdbcSourceConnectorConstants.TABLE_NAME_KEY, SINGLE_TABLE_NAME);
+    SINGLE_TABLE_PARTITION_WITH_VERSION.put(
+        JdbcSourceConnectorConstants.TABLE_NAME_KEY,
+        SINGLE_TABLE_NAME);
+    SINGLE_TABLE_PARTITION_WITH_VERSION.put(
+        JdbcSourceConnectorConstants.OFFSET_PROTOCOL_VERSION,
+        "1");
```

Comment:
This should all be moved into the PartitionProtocols class, which can have static instances of the strategy for each protocol.

Reply:
style nit: the `);` should be on the next line.